msg/SystemVersionString.msg

```diff
@@ -5,5 +5,5 @@ char[32] hw_version		# main autopilot hw version
 char[32] sw_version		# main autopilot sw version
 char[32] os_version		# OS or middleware version
 char[32] bl_version		# bootloader version
-char[32] component_version1	# vendor specific component version
```

Comment:
This looks sensible, but I wouldn't do this still. I believe it requires re-compiling the linux-side uXRCE-DDS type definitions for the mission computer to match the definition. Maybe we could reduce the size of the hw_info and fpga_info and print out the accelerator(FPGA) version in a bit shorter form (e.g. remove the FPGA: string) to fit into 32 bytes?

Reply:
Right. I'll leave this as it was.